### DISCUSSION TITLE
A completely open & reproducible version of the SPOC Verifier?

### DISCUSSION THREAD

**Discussion by amandawhitmire** [sul-dlss-labs/spoc#40] [Ideas]
A challenge that we face in sharing the outputs of our first SPOC spike process is that most of the student papers and the entirety of the WoRMS taxonomy database cannot be shared alongside the Streamlit app, due to permissions limitations. I'm wondering if we can share a subset of the student papers as a test set (say, the UW papers?), and maybe some other genus/species taxonomy that is openly available (I'll look into it?). Is there interest in offering this alongside the app, code and documentation? If yes, I can assemble the content for review.

**Comment by cncoleman** [comment]
@amandawhitmire Now that you have permission to share some of the documents, I think that would be helpful. I think of our formal outputs as non-consumptive use of the papers and so not restricted by permission to share access to the full document. This opens important questions about how we share or don't share the full text as documentation of our training process.

**Reply by amandawhitmire** [reply]
Yes, exactly. The text/data outputs can be fully open, but if we wanted to share some sample papers and entities alongside the Streamlit app for others to test out, we need to create them.
